plugins/modules/win_domain.ps1

```diff
@@ -9,13 +9,26 @@ $ErrorActionPreference = "Stop"
 
 # FUTURE: Consider action wrapper to manage reboots and credential changes
 
+$required_features = @("AD-Domain-Services","RSAT-ADDS", "RSAT-AD-AdminCenter")
+
+Function Get-MissingFeatures {
+    $features = @(Get-WindowsFeature $required_features)
+
+    If($features.Count -ne $required_features.Count) {
+        Throw "One or more Windows features required for a domain controller are unavailable"
```

Comment:
Would be good to get the list of features that are missing to help with future debugging.

Reply:
Added the list of unavailable features to the thrown message.